app/diagnostics.go

```diff
@@ -593,7 +602,7 @@ func (a *App) trackConfig() {
 		"trace":                             *cfg.ElasticsearchSettings.Trace,
 	})
 
-	a.SendDiagnostic(TRACK_CONFIG_PLUGIN, map[string]interface{}{
+	pluginConfigData := map[string]interface{}{
```

Comment:
I'm wondering if we should rely `PrepackagedPlugins()` that will get added with https://github.com/mattermost/mattermost-server/pull/13449/ and not hard code this list.

Reply:
In the efforts to keep this PR straightforward and in time for v5.20, I've included a ticket to do that change when we add all marketplace plugins for telemetry - see [MM-21685](https://mattermost.atlassian.net/browse/MM-21685)